### PR TITLE
Add docs for automated k3s upgrades

### PR DESCRIPTION
**Pull request by cjellick**

### CODE REVIEW

File: content/k3s/latest/en/upgrades/automated/_index.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+weight: 20
+---
+
+>**Note:** This feature is available as of [v1.17.4+k3s1](https://github.com/rancher/k3s/releases/tag/v1.17.4%2Bk3s1)
+
+### Overview
+
+You can manage K3s cluster upgrades using Rancher's system-upgrade-controller. This is a Kubernetes-native approach to cluster upgrades. It leverages a [custom resource definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources), the `plan`, and a [controller](https://kubernetes.io/docs/concepts/architecture/controller/) that schedules upgrades based on the configured plans.
+
+A plan defines upgrade policies and requirements. This documentation will provide plans with defaults appropriate for upgrading a K3s cluster. For more advanced plan configuration options, please review the [CRD](https://github.com/rancher/system-upgrade-controller/blob/master/pkg/apis/upgrade.cattle.io/v1/types.go).
```
Comment (davidnuzik) [inline content/k3s/latest/en/upgrades/automated/_index.md]: Nit: Is there a better way to describe the advanced plan config options? Or perhaps longer term they should be documented better somehow? types.go is readable here but not an ideal way to visualize all the options IMHO.

Comment (cjellick) [reply content/k3s/latest/en/upgrades/automated/_index.md]: yeah, its not ideal, but the best we can do for this release I think.

File: content/k3s/latest/en/upgrades/automated/_index.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+To automate upgrades in this manner you must:
+
+1. Install the system-upgrade-controller into your cluster
+1. Configure plans
+
+
+### Install the system-upgrade-controller
+The system-upgrade-controller can be installed as a deployment into your cluster. The deployment requires a service-account, clusterRoleBinding, and a configmap. To install these components, run the following command:
+```
+kubectl apply -f https://raw.githubusercontent.com/rancher/system-upgrade-controller/master/manifests/system-upgrade-controller.yaml
```
Comment (davidnuzik) [inline content/k3s/latest/en/upgrades/automated/_index.md]: Question: is it generally safe to always use master ? What if we make a breaking change here? SLA implications, etc?

Comment (cjellick) [reply content/k3s/latest/en/upgrades/automated/_index.md]: yeah, another good catch...The plan is for Jacob to pusblish the yaml as a release artifact. when he does, we'll update this. he should have that done tonight or tomorrow

Comment (davidnuzik) [reply content/k3s/latest/en/upgrades/automated/_index.md]: SGTM

**Pull request merged**